<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finance Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="menuFrame">
      <layout class="QVBoxLayout" name="menuLayout">
       <item>
        <widget class="QPushButton" name="btnManagement">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Management</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFinance">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Finance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnContentCreator">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Content Creator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSponsor">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Sponsor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeal">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Deal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEmployee">
         <property name="styleSheet">
          <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Employee</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="contentLayout">
      <item>
       <layout class="QHBoxLayout" name="topBar">
        <item>
         <widget class="QLineEdit" name="searchBox">
          <property name="styleSheet">
           <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
          </property>
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearchById">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Rechercher</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDarkTheme">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Dark Theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSortByMontant">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Trier par Montant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExportCSV">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Export to CSV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="formBox">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;
        border: 1px solid #aaa;
        border-radius: 6px;
        margin-top: 10px;</string>
        </property>
        <property name="title">
         <string>Manage Finance - Factures</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelId">
           <property name="text">
            <string>ID Facture :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineId">
           <property name="styleSheet">
            <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelMontant">
           <property name="text">
            <string>Montant :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineMontant">
           <property name="styleSheet">
            <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDateEmission">
           <property name="text">
            <string>Date émission :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="dateEmission">
           <property name="styleSheet">
            <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelDateEcheance">
           <property name="text">
            <string>Date échéance :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="dateEcheance">
           <property name="styleSheet">
            <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelStatut">
           <property name="text">
            <string>Statut :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboStatut">
           <property name="styleSheet">
            <string notr="true">  border: 1px solid #ccc;
        border-radius: 5px;
        padding: 4px;
        background: #f9f9f9;</string>
           </property>
           <item>
            <property name="text">
             <string>Payée</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En attente</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Annulée</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionLayout">
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="styleSheet">
           <string notr="true">background-color: #6c63ff;
        color: white;
        border-radius: 8px;
        padding: 6px 12px;
        font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Clear Form</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableFactures">
        <property name="styleSheet">
         <string notr="true"> border: 1px solid #ccc;
        gridline-color: #ddd;
        selection-background-color: #6c63ff;
        selection-color: white;</string>
        </property>
        <column>
         <property name="text">
          <string>ID Facture</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date émission</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date échéance</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Statut</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
